modules/integration_azure-storage-account/conf/06-requests-rate-status.yaml

```diff
@@ -0,0 +1,25 @@
+module: "Azure Storage Account"
```

Comment:
having a global requests status detector does not make sense in my opinion (we have removed it from our previous Datadog implementation). We should monitor the failed request rate per usage (blob, table ...).